docs/people-and-groups/managing.md

```diff
@@ -113,6 +113,28 @@ This action will delete any dashboard subscriptions or alerts the person has cre
 
 This action doesn't affect email distribution lists that are managed outside of Metabase.
 
+## Default user accounts
+
+Metabase includes default user accounts to handle various activities. Metabase will exclude these user accounts from the **Admin settings** > **People** tab, so we're documenting the accounts here so you know they're legitimate accounts and not someone trying to spy on your Metabase.
+
+### Anonymous user account
```

Comment:
I'm not sure if we need to explain this, but this user doesn't even exist in the appdb.

It is a virtual user, created in Metabase analytics views.